plugins/meta/isolation/main.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2017 CNI authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/containernetworking/cni/pkg/skel"
+	"github.com/containernetworking/cni/pkg/types"
+	"github.com/containernetworking/cni/pkg/types/current"
+	"github.com/containernetworking/cni/pkg/version"
+	"github.com/coreos/go-iptables/iptables"
+
+	bv "github.com/containernetworking/plugins/pkg/utils/buildversion"
+)
+
+// PluginName
+const PluginName = "isolation"
+
+// PluginConf
+type PluginConf struct {
+	types.NetConf
+
+	// Chained prev results
+	RawPrevResult *map[string]interface{} `json:"prevResult"`
+	PrevResult    *current.Result         `json:"-"`
+
+	// Internal states
+	bridgeName string
```

Comment:
If this field is internal, an ignored json tag `json:"-"` will be needed

Reply:
`go vet` says no:

```console
suda@suda-ws01:~/gopath/src/github.com/containernetworking/plugins/plugins/meta/isolation$ git diff 
diff --git a/plugins/meta/isolation/main.go b/plugins/meta/isolation/main.go
index dd274ae..efb0b98 100644
--- a/plugins/meta/isolation/main.go
+++ b/plugins/meta/isolation/main.go
@@ -39,7 +39,7 @@ type PluginConf struct {
        PrevResult    *current.Result         `json:"-"`
 
        // Internal states
-       bridgeName string
+       bridgeName string `json:"-"`
 }
 
 // parseConfig parses the supplied configuration (and prevResult) from stdin.
suda@suda-ws01:~/gopath/src/github.com/containernetworking/plugins/plugins/meta/isolation$ go vet .
# github.com/containernetworking/plugins/plugins/meta/isolation
./main.go:42:2: struct field bridgeName has json tag but is not exported
```